src/template-tags/tickets.php

```diff
@@ -109,6 +109,164 @@ function tribe_events_count_available_tickets( $event = null ) {
 	}
 }//end if
 
+if ( ! function_exists( 'tribe_events_count_tickets_and_rsvp' ) ) {
+
+	/**
+	 * Returns Ticket and RSVP Count for an Event
+	 *
+	 * @param $event_id
+	 *
+	 * @return array
+	 */
+	function tribe_events_count_tickets_and_rsvp( $event_id ) {
```

Comment:
Is there a directory to place the class in? I was looking in Event Tickets and was not clear on where to put it.